Jenkinsfile

```diff
@@ -146,7 +147,9 @@ ansiColor('xterm') {
 
             archive 'packages/node_modules/@ciscospark/widget-message-meet/dist/**/*'
             archive 'packages/node_modules/@ciscospark/widget-space/dist/**/*'
-            archive 'dist/**/*'
+            archive 'packages/node_modules/@ciscospark/widget-message-meet-demo/dist/**/*'
```

Comment:
@bzang I'm 99% confident that the raw dist folder was for the demo, but if not, let me know!